examples/cluster/python/__main__.py

```diff
@@ -95,7 +95,7 @@ def wait_for_ready(private_ip):
     # time to spin up and be ready
     # if we had a custom image with docker installed then it would
     # be a faster start up time
-    time.sleep(120)
+    time.sleep(240)
```

Comment:
creating a thread here to debug nodejs test failure i'm seeing [here](https://github.com/pulumi/pulumi-rke/actions/runs/4070063399/jobs/7010589966#step:22:413)
> Failed running cluster err:[workerPlane] Failed to bring up Worker Plane: [Failed to verify healthcheck: Failed to check http://localhost:10248/healthz for service [kubelet] on host [34.221.48.176]: Get "http://localhost:10248/healthz": Unable to access the service on localhost:10248. The service might be still starting up. Error: ssh: rejected: connect failed (Connection refused), log: Error: failed to run Kubelet: unable to determine runtime API version: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial unix: missing address"]

cc @aq17 @AaronFriel